sequence_check_digit/README.rst

```diff
@@ -0,0 +1,61 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Sequence Check Digit
+====================
+
+This module was written to configure check digits on sequences added on the end.
+It is useful as a control of the number on visual validation.
+
+It is useful when some manual checks are required or on integrations.
+The implemented codes can avoid modification of one character and flip of
+two consecutive characters.
+
+
+Configuration
+=============
+
+No manual configuration is needed.
+
+Usage
+=====
+
+* Access sequences and configurate the model to use.
+* The model will check if the format of prefix, suffix and number is valid
+
+Implemented algorithms
+======================
```

Comment:
remove `====================`. Instead, you can write `*Implemented algorithms:`